hw/bsp/espressif/components/tinyusb_src/CMakeLists.txt

```diff
@@ -15,6 +15,13 @@ list(APPEND compile_definitions
   BOARD_TUH_MAX_SPEED=${RHPORT_HOST_SPEED}
   )
 
+if (target STREQUAL esp32p4)
+  # P4 change alignment to 64 (DCache line size) for possible DMA configuration
+  list(APPEND compile_definitions
+    CFG_TUSB_MEM_ALIGN=__attribute__\(\(aligned\(64\)\)\)
```

Comment:
CFG_TUSB_MEM_ALIGN will cause class driver to put correct aligment on usb/dma buffer.

Reply:
@roma-jam I just realized that we probably does not need the dma buffer to be 64 bytes aligned ? It only need to occupies the whole cache line (64 byte) right ? If you we can change it back to simply 4 bytes alignment.

Reply:
Unfortunately the cache line size is configurable, and can be 64 or 128 bytes :( 
This might be good enough for now, just so you keep this in mind if someone reports issues...

Reply:
what is the default value ? it isn't an problem at all. both the CFG_TUSB_MEM_ALIGN and CFG_TUD_MEM_DCACHE_LINE_SIZE is configurable at user level (tusb_config.h). So if use change it to 128, they shoud set this accordingly.

Reply:
default is 64. OK, great!